src/wp-includes/html-api/class-wp-html-processor.php

```diff
@@ -2235,11 +2234,12 @@ private function step_in_table(): bool {
 			 */
 			case '+COL':
 				$this->state->stack_of_open_elements->clear_to_table_context();
+
 				/*
 				 * > Insert an HTML element for a "colgroup" start tag token with no attributes,
 				 * > then switch the insertion mode to "in column group".
 				 */
-				$this->insert_html_element( new WP_HTML_Token( null, 'COLGROUP', false ) );
+				$this->insert_virtual_node( 'COLGROUP' );
```

Comment:
When I got into this @sirreal I preferred giving a bookmark to the virtual nodes, letting people return to them. Once I did that there was a reasonable amount of ceremony around creating and inserting the token, so I went ahead and created this helper method. Please let me know if you object to it.

Reply:
I like the new method 👍